jetson/percep/pcl.cpp

```diff
@@ -28,231 +29,244 @@ void PCL::PassThroughFilter(const std::string axis, const double upperLimit) {
 //All points in a cluster are then reduced to a single point
 //This point is the centroid of the cluster
 //Source: https://rb.gy/2ybg8n
-void PCL::DownsampleVoxelFilter() {
-    #if PERCEPTION_DEBUG
-        pcl::ScopeTime t ("VoxelFilter");
-    #endif
+void PCL::DownsampleVoxelFilter()
+{
+#if PERCEPTION_DEBUG
+    pcl::ScopeTime t("VoxelFilter");
+#endif
 
     pcl::VoxelGrid<pcl::PointXYZRGB> sor;
-    sor.setInputCloud (pt_cloud_ptr);
+    sor.setInputCloud(pt_cloud_ptr);
     sor.setLeafSize(20.0f, 20.0f, 20.0f);
-    sor.filter (*pt_cloud_ptr);
+    sor.filter(*pt_cloud_ptr);
 }
 
 /* --- RANSAC Plane Segmentation Blue --- */
 //Picks three random points in point cloud
 //Counts how many points lie on or near the plane made by these three
-//If the number of points in the plane (score) is greater than 
+//If the number of points in the plane (score) is greater than
 //some threshold then a valid plane has been found
 //Colors all points in this plane blue or
 //removes points completely from point cloud
 //Source: https://rb.gy/zx6ojh
-void PCL::RANSACSegmentation(string type) {
-    #if PERCEPTION_DEBUG
-        pcl::ScopeTime t ("RANSACSegmentation");
-    #endif
+void PCL::RANSACSegmentation(string type)
+{
+#if PERCEPTION_DEBUG
+    pcl::ScopeTime t("RANSACSegmentation");
+#endif
 
     //Creates instance of RANSAC Algorithm
     pcl::SACSegmentation<pcl::PointXYZRGB> seg;
     seg.setOptimizeCoefficients(true);
-    seg.setModelType(pcl::SACMODEL_PERPENDICULAR_PLANE );
+    seg.setModelType(pcl::SACMODEL_PERPENDICULAR_PLANE);
     seg.setMethodType(pcl::SAC_RANSAC);
     seg.setMaxIterations(400);
     seg.setDistanceThreshold(100); //Distance in mm away from actual plane a point can be
     // to be considered an inlier
     seg.setAxis(Eigen::Vector3f(0, 1, 0)); //Looks for a plane along the Z axis
-    double segmentation_epsilon = 10; //Max degree the normal of plane can be from Z axis
+    double segmentation_epsilon = 10;      //Max degree the normal of plane can be from Z axis
     seg.setEpsAngle(pcl::deg2rad(segmentation_epsilon));
 
     //Objects where segmented plane is stored
     pcl::ModelCoefficients::Ptr coefficients(new pcl::ModelCoefficients());
     pcl::PointIndices::Ptr inliers(new pcl::PointIndices());
-    
+
     seg.setInputCloud(pt_cloud_ptr);
     seg.segment(*inliers, *coefficients);
 
-    if(type == "blue") {
-        for(int i = 0; i < (int)inliers->indices.size(); i++) {
-        pt_cloud_ptr->points[inliers->indices[i]].r = 255;
-        pt_cloud_ptr->points[inliers->indices[i]].g = 255;
-        pt_cloud_ptr->points[inliers->indices[i]].b = 0;
+    if (type == "blue")
```

Comment:
Going forward we aren't going to have spaces between () and the keyword in if statements, function names, and for loops. If you could change these back throughout the code that'd be great. We have a separate Issue #516 where we will go through and clean up a lot of the syntactical inconsistencies in our code. I'll also make a best practices guide and can send that to everyone on Sunday

Reply:
I'd rather this PR only contain changes relating to fixing our silent mode, and not changes aimed at cleaning up our overall code

Reply:
Fixed style in our sections of code